### DISCUSSION TITLE
Add faroese (fo) language to directus

### DISCUSSION THREAD

**Discussion by Horib** [directus/directus#10719] [Feature Requests]
I was directed by Jonathan to open a discussion on github to get faroese added to directus

"Please open a GH ticket to get the language added - https://github.com/directus/directus/discussions/new

Engineering enabled the language mod so you can start translations as well:
https://locales.directus.io/project/directus/fo"

**Comment by rijkvanzanten** [comment]
Beat you to it https://github.com/directus/directus/commit/54ec5b2436c4137dadb10b685265338c36141931 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
